src/components/NetworkService.ts

```diff
@@ -12,7 +12,8 @@ const logger = log.extend( "NetworkService" );
 
 const NetworkService = ( ) => {
   const { loadTime } = usePerformance( {
-    screenName: "NetworkService"
+    screenName: "NetworkService",
+    isLoading: false
```

Comment:
`isLoading` is a required parameter. Previously implicitly `undefined` now explicitly `false`